src/Api/AddressComplete/AddressCompleteApi.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Eckinox\AddressBundle\Api\AddressComplete;
+
+use Eckinox\AddressBundle\Api\AddressApiInterface;
+use Eckinox\AddressBundle\Api\AddressComplete\Model\Address;
+use Eckinox\AddressBundle\Api\AddressComplete\Model\Prediction;
+use Psr\Log\LoggerInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class AddressCompleteApi implements AddressApiInterface
+{
+	private HttpClientInterface $client;
+
+	private string $language;
+
+	public function __construct(
+		HttpClientInterface $addressComplete,
+		private LoggerInterface $logger,
+		private string $apiKey,
+	) {
+		$this->client = $addressComplete;
+		$this->language = "FRE"; // this should also be based on config
```

Comment:
As you said yourself: this should also be based on a config :p

Reply:
little guy slipped through the cracks, fixed it !